### PR TITLE
add try and exception

### PR DESCRIPTION
**Pull request by MikeLing**

### CODE REVIEW

File: tools/failures.py [hunk 1]
```diff
@@ -68,65 +74,6 @@ def communicate(failures, to_insert, total_detected, testmode, date):
                    " changes from previous day", change, admin=True, results=True)
 
 
-def format_in_table(active_jobs, master):
```
Comment (MikeLing) [inline tools/failures.py]: I remember we had remove it before, due to  we are storing the high value jobs now :)

**Comment by MikeLing** [comment]
@jmaher r?

**Comment by jmaher** [comment]
I will look at this on Monday.

Joel

**Comment by jmaher** [comment]
great stuff!

**Pull request merged**